tests/governance_js.py

```diff
@@ -69,9 +69,24 @@ def set_service_recent_cose_proposals_window_size(proposal_count):
     )
 
 
+def choose_node(network):
+    # Ideally, this would use find_random_node - you should be able to use any
+    # node for governance.
+    # However, many of these tests include a pattern of
+    #   POST /proposal
+    #   GET /proposal
+    # If the former request is redirected, then the latter may fail (essentially
+    # reading stale state, assuming session consistency that doesn't exist).
+    # return network.find_random_node()
```

Comment:
This may be a good motivation to prototype ETag-based concurrency in the governance (at first glance, the proposalId would make a decent choice).

Reply:
Good suggestion. Opened an issue to track this: https://github.com/microsoft/CCF/issues/6081